MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/Models/ListItem.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Web;
+
+namespace MyPerfectOnboarding.Api.Models
+{
+    [DataContract]
+    public class ListItem
+    {
+        private Guid _id;
+        private string _text;
+        private bool _isActive;
+        private DateTime _creationTime;
+        private DateTime _lastUpdateTime;
+
+        public ListItem(Guid id, string text, bool isActive, DateTime creationTime, DateTime lastUpdateTime)
```

Comment:
I will leave it be this time, but I am sure you wouldn't need the constructor in the further Tasks 😉.